src/filesystem/impls/filer/FilerFileSystem.js

```diff
@@ -193,11 +193,15 @@ define(function (require, exports, module) {
                     BlobUtils.rename(oldPath, newPath);
                     BrambleEvents.triggerFileRenamed(oldPath, newPath);
                 }
-
-                callback();
+                FileSystemCache.refresh(function(){
+                  if(err){    
+                    return callback(err);
+                  }
+                    LiveDevMultiBrowser.reload();
+                    callback();
+                })();
```

Comment:
This should just be `});`, which will a) close the callback function you started on 196; and b) close the invocation of the `refresh()` method.  You don't need a further invocation.